core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -107,4 +120,19 @@ public static boolean ancestorOf(Table table, long snapshotId, long ancestorSnap
 
     return newFiles;
   }
+
+  /**
+   * traverses through the table's snapshots and finds the {@link Table}'s
+   * Snapshot that is committed after the specified snapshotId
+   * @return null if passed in snapshot expired, else the snapshot after the passed in snapshot
+   */
+  public static Snapshot snapshotAfter(Table table, long snapshotId) {
+    Snapshot previousSnapshot = table.snapshot(snapshotId);
+    Snapshot pointer = table.currentSnapshot();
```

Comment:
Nit: other methods in this file use `current` instead of `pointer`. It would be good to rename.

Reply:
done.